_docs/kustomize/akash-provider/deployment.yaml

```diff
@@ -26,50 +26,50 @@ spec:
             #
             # key.txt:      exported wallet key
             # key-pass.txt: password used to encrypt exported key
-            - name: AKASHCTL_BOOT_KEYS
+            - name: AKASH_BOOT_KEYS
```

Comment:
![](https://media0.giphy.com/media/l3q2XhfQ8oCkm1Ts4/giphy.gif?cid=ecf05e47wbok4eihd0k98qeqq3r1io8c1t7atnx6l9h1x0em&rid=giphy.gif)

hooray for naming sanity!